src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const docPopulationArray = document.querySelectorAll('.population');
+
+const docTotal = document.querySelector('.total-population');
+
+const docAverage = document.querySelector('.average-population');
+
+const populationNumbers = [];
+
+docPopulationArray.forEach((item) => {
+  populationNumbers.push(parseInt(item.textContent.replaceAll(',', '')));
+});
```

Comment:
In JavaScript, the `map()` method is a built-in function commonly used to iterate over an array, applying a function to each element and returning a new array with the results. It does not modify the original array but instead creates a new one. However, it's not necessary to create a new array with elements parsed to numbers using `map()`. You can achieve this within the `reduce()` function as well.